.github/workflows/main.yml

```diff
@@ -220,6 +220,23 @@ jobs:
               cpython: "yes",
               cpython_arg: "--host=armv5-unknown-linux-musleabi --target=armv5-unknown-linux-musleabi",
             }
+          - {
+              image: "linux-armv5-uclibc",
+              stockfish: "no",
+              stockfish_arg: "",
+              ninja: "yes",
+              ninja_arg: "",
+              openssl: "yes",
+              openssl_arg: "linux-armv4",
+              C: "yes",
```

Comment:
You can disable this C tests: https://github.com/dockcross/dockcross/runs/5028644324?check_suite_focus=true#step:10:232